model.py

```diff
@@ -44,14 +44,14 @@ def from_name(cls, name: str):
         if name in transformer_configs:
             return cls(**transformer_configs[name])
         # fuzzy search
-        config = [config for config in transformer_configs if config in str(name).upper() or config in str(name)]
+        config = [config for config in transformer_configs if config.lower() in str(name).lower() or config.lower() in str(name).lower()]
```

Comment:
Both condition in "if" statement are the same.
if **config.lower() in str(name).lower()** or **config.lower() in str(name).lower()**

Reply:
Fixed